lib/scope.js

```diff
@@ -54,7 +54,33 @@ let is_secret_var = function(row) {
   return original.flags !== undefined && original.flags.secret_name !== undefined;
 }
 
+let fromBase64 = function(base64_str) {
+  /* Takes a base64 encoded string and returns it decoded.
+   * Note that this accept take strings with invalid base 64 characters,
```

Comment:
Very good stuff to know!

```suggestion
   * Note that this can take strings with invalid base 64 characters,
```